minecraft-server/start-finalSetup05EnvVariables

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+if [ "$REPLACE_ENV_VARIABLES" = "TRUE" ]; then
+  echo "Replacing env variables in configs that match the prefix $ENV_VARIABLE_PREFIX..."
+  while IFS='=' read -r name value ; do
+    # check if name of env variable matches the prefix
+    # sanity check environment variables to avoid code injections
+    if [[ "$name" = $ENV_VARIABLE_PREFIX* ]] && [[ $value =~ ^[0-9a-zA-Z_:/=?.+\-]*$ ]] && [[ $name =~ ^[0-9a-zA-Z_\-]*$ ]]; then
+        echo "$name = $value"
+        find /data/ -type f -exec sed -i 's#${'"$name"'}#'"$value"'#g' {} \;
```

Comment:
It looks this `find` needs to include only common filename patterns or exclude jar (and other binary files, like `.db`):

When restarting the container with an already populated data volume, I got:
```
Replacing env variables in configs that match the prefix CFG_...
CFG_MINE = testing 
sed: can't move '/data/minecraft_server.1.13.2.jarDaJAED' to '/data/minecraft_server.1.13.2.jar': No such file or directory 
```

Reply:
Yes you are right. Do you think the following file types would be enough:

- `.yml`
- `.yaml`
- `.properties`
- `.cfg`
- `.txt`

Reply:
Yep, that seems to cover it.